### PR TITLE
[macOS] move Pester tests to separate files/contexts/describes

### PR DESCRIPTION
**Pull request by Darleev**
# Description
Improvement
Need to move tests according to other environments in separate folder and contexts

#### Related issue:
https://github.com/actions/virtual-environments-internal/issues/1710

## Check list
- [+] Related issue / work item is attached
- [+] Tests are written (if applicable)

### CODE REVIEW

**Review by vsafonkin — APPROVED** [review]

File: images/macos/tests/Powershell.Tests.ps1 [hunk 1]
```diff
@@ -38,4 +40,10 @@ Describe "Powershell" {
             }
         }
     }
+    
+    Context "Azure CLI" {
```
Comment (al-cheb) [inline images/macos/tests/Powershell.Tests.ps1:44]: Why is az cli related to PowerShell section?

File: images/linux/scripts/installers/basic.sh [hunk 2]
```diff
@@ -12,4 +12,4 @@ for package in $common_packages $cmd_packages; do
     apt-get install -y --no-install-recommends $package
 done
 
-invoke_tests "Apt"
\ No newline at end of file
+invoke_tests "BasicTools"
```
Comment (al-cheb) [inline images/linux/scripts/installers/basic.sh:15]: Why do we change linux scripts?

File: images/macos/tests/Powershell.Tests.ps1 (hunk 1 above)
Comment (Darleev) [reply images/macos/tests/Powershell.Tests.ps1:44]: Az CLI installs in powershell.sh script. I think it's the best option to run it together with powershell itself.

File: images/linux/scripts/installers/basic.sh (hunk 2 above)
Comment (Darleev) [reply images/linux/scripts/installers/basic.sh]: Thank you, very good point. I've fixed it.

**Review by dmitry-shibanov — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Pull request merged**